queries/liquidsoap/highlights.scm

```diff
@@ -8,6 +8,8 @@
   "def_end"
 ] @keyword.function
 
+(let_decoration) @keyword.coroutine
```

Comment:
doesn't look right to me but I could be wrong

Reply:
Naming isn't great but it's correct.